File: DAM_1er_año/Programacion/Python/Tema 1/Pycharm/ejercicio1/ej4.py
tabla_desde = 0
tabla_hasta = 10
desde = 0
hasta = 10

for factor1 in range(tabla_desde, tabla_hasta + 1):
	print(f'Tabla de multiplicar del {factor1}:')
	for factor2 in range(desde, hasta + 1):
		print(f'{factor1} x {factor2} = {factor1 * factor2}')
	print()

File: DAM_2º-año_1º/Programación/UD4-Ej1/src/Ej4.java
import java.util.Scanner;

/**
 * Crear una función "EscribirTablaMultiplicar", que reciba como parámetro un
 * número entero, y escriba la tabla de multiplicar de ese número (por ejemplo,
 * para el 3 deberá llegar desde "3x1=3" hasta "3x10=30").
 * **/
public class Ej4 {
    static void EscribirTablaMultiplicar(int n){
        for (int i = 0; i < 11; i++) {
            int multi = n * i;
            System.out.println(n + "x" + i + "=" + multi);
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Que tabla de multiplicar quieres saber: ");
        int num = scanner.nextInt();

        EscribirTablaMultiplicar(num);
    }

}
